atom/kite/lib/kite.js

```diff
@@ -349,4 +338,15 @@ module.exports = {
     // focus is tracked at the workspace level.
     atom.workspace.onDidChangeActivePaneItem(this.outgoing.onFocus.bind(this.outgoing));
   },
+  completions: function() {
+    return require('./completions.js');
```

Comment:
can we require this at the beginning of the file, store it in a variable and have this function just return that variable?